tools/imagebuilder/terraform/main.tf

```diff
@@ -123,5 +123,10 @@ resource "null_resource" "copy_os_image" {
   provisioner "local-exec" {
     command = "scp -r ${var.vm_user}@${data.nutanix_virtual_machine.build_vm_datasource.nic_list.0.ip_endpoint_list[0].ip}:~/image-builder/images/capi/output ."
   }
+
```

Comment:
is it nor part of another PR ???

Reply:
Indeed, this got merge in https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/131

Reply:
issues with rebasing.